1-run-automation-test.md

```diff
@@ -0,0 +1,126 @@
+# Running Automation test with Kobiton
+This guide will demonstrate how to run an automation test with Kobiton on Teamcity.
+## Table of contents
+[1. Setup TeamCity](#1-setup-teamcity)
+
+[2. Prepare Kobiton configuration for executing automation testing](#2-prepare-kobiton-configuration-for-executing-automation-testing)
+
+[3. Write the automation test script](#3-write-the-automation-test-script)
+
+[4. Configure your TeamCity project for Kobiton testing](#4-configure-your-teamcity-project-for-kobiton-testing)
+
+[5. Call Kobiton REST API to get session information](#5-call-kobiton-rest-api-to-get-session-information)
+## Prerequisites 
+- Java(JRE). Supported are:
+  + Oracle java 8 and updates
+  + OpenJDK 8
+- Kobiton account
+## 1. Setup TeamCity
+This part will guide you setup TeamCity and configure TeamCity with GitHub. Skip step if you already have TeamCity setup.
+ + You can download TeamCity from [here](https://www.jetbrains.com/teamcity/download/).  
+ + For introduce on how to install and configure the TeamCity server, follow [this guide](https://confluence.jetbrains.com/display/TCD18/Installing+and+Configuring+the+TeamCity+Server).
+
+Set up your project and link your app repository that we will test with Kobiton later.
++ Connect GitHub with TeamCity, you can follow [this guide](https://confluence.jetbrains.com/display/TCD10/Integrating+TeamCity+with+VCS+Hosting+Services).
+ 
++ More information about TeamCity can be found in [this tutorial](https://confluence.jetbrains.com/display/TCD10/TeamCity+Documentation).
+
+## 2. Prepare Kobiton configuration for executing automation testing 
+If you do not have a Kobiton account yet, go ahead to [create a free trial account](https://kobiton.com/freetrial/) and sign in. It takes just a few moments.
+
+ **a. Get Kobiton Username**
+  + Go to  https://portal.kobiton.com
+  + In the upper right hand corner, click on User icon and in the drop down menu, select 'Profile'.
+
+ ![alt text](./assets/click_profile.png )
+ 
+ + You should see your username.
+ 
+ ![alt text](./assets/username.png )
+
+ **b. Get Kobiton API key**
++ Click on user icon in the upper right hand corner again and select 'Setting'.
+
+ ![alt text](./assets/click_setting.png )
+
+ + You will see your API key under the 'API Keys'. 
+
+ ![alt text](./assets/api_key.png )
+
+## 3. Write the automation test script
+**a. Get Desired cap**
```

Comment:
I think that "desiredCaps" should be used instead of "desired cap", so as to match the variable in the provided js example.